lib/game_team_list.rb

```diff
@@ -2,17 +2,19 @@
 require_relative './game_team'
 
 class GameTeamList
-  attr_reader :game_teams
+  attr_reader :game_teams, :stat_tracker
 
   def initialize(path, stat_tracker)
-      @game_teams = create_game_teams(path)
-  end
-
-  def create_game_teams(path)
-      data = CSV.parse(File.read(path), headers: true, header_converters: :symbol)
-      data.map do |datum|
-        GameTeam.new(datum, self)
-      end
+    @stat_tracker = stat_tracker
+    @game_teams = create_game_team_list(path)
   end
 
+    def create_game_team_list(path)
+        arr = []
```

Comment:
Since you're using map, I don't think you need to define an "arr" variable here. I think it can just be:

```
def create_game_team_list(path)
  data = CSV.parse(File.read(path), headers: true, header_converters: :symbol)
    data.map do |datum|
      GameTeam.new(datum, self)
    end
end
```